development/tools/jobs/kyma/application_broker_test.go

```diff
@@ -22,7 +22,14 @@ func TestApplicationBrokerReleases(t *testing.T) {
 			tester.AssertThatHasExtraRefTestInfra(t, actualPresubmit.JobBase.UtilityConfig, currentRelease)
 			tester.AssertThatHasPresets(t, actualPresubmit.JobBase, tester.PresetDindEnabled, tester.PresetDockerPushRepo, tester.PresetGcrPush, tester.PresetBuildRelease)
 			assert.True(t, actualPresubmit.AlwaysRun)
-			tester.AssertThatExecGolangBuildpack(t, actualPresubmit.JobBase, tester.ImageGolangBuildpackLatest, "/home/prow/go/src/github.com/kyma-project/kyma/components/application-broker")
+
+			var imgVer string
+			if tester.HasOneOfSuffixes(currentRelease, "-0.7", "-0.8", "-0.9") {
```

Comment:
I know that you do not introduce this method HasOneOfSuffixes, but maybe we should fix it now:
- I already found such code:
```
f tester.HasOneOfSuffixes(relBranch, "-0.6", "-0.7", "-0.8") 
```
and 0.6 is no longer supported. Maybe we should just define in the tester package constants  for supported release branches:
```
const (
	rel07 = "release-0.7"
	rel08 = "release-0.8"
	rel09 = "release-0.9"

)

// GetAllKymaReleaseBranches returns all supported kyma release branches
func GetAllKymaReleaseBranches() []string {
	return []string{rel07,rel08, rel09}
}
```
Then, if we decide to not support 0.7, we remove `rel07` const and then look for places where it was previously used. And maybe we can remove completely `HasOneOfSuffiexes` ?